gridt/src/app/home/home.page.html

```diff
@@ -27,8 +27,8 @@
 
   <!-- Can be rewritten as an if/else statement together with previous if-->
   <ion-grid *ngIf="movements$ && (movements$ | async).length > 0">
-    <ion-col>
-      <ion-row>
```

Comment:
By changing the order each card is now full column width. This is fine for mobile devices but does look a bit awkward on larger screens. Is there something you can do to improve the visuals of the cards on medium to large screens?

Reply:
If you mean the cards themselves then i think a new issue is in order as that falls outside of the scope of this issue i think.
Edit: The cards themselves are not of equal size when the amount of leaders differ.

Reply:
If you mean displaying multiple items in one row, than that can be realized by moving the ngFor to the col instead of the item. This fixes that somewhat while still working for smaller devices. The problem i am now encountering is that the offset is getting repeated.